Dockerfile.build

```diff
@@ -0,0 +1,17 @@
+FROM debian:8
+
+WORKDIR /code
+ADD . /code
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    imagemagick \
+    libmagickwand-dev \
+    curl
+RUN \curl -L https://get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm install 2.4.1"
```

Comment:
don't need `/bin/bash` here

Reply:
for some reason, when I don't include /bin/bash it doesn't work, says rvm isn't found. I feel like it has to do with the user/group created during rvm install